trie/node_extension.go

```diff
@@ -22,3 +24,14 @@ func shouldTestNode(n node, key []byte) bool {
 
 	return false
 }
+
+func snapshotGetTestPoint(key []byte, faultyChance int) error {
```

Comment:
we have the `unused function` linter issue. add a dummy test that calls this function so we can get rid of the linter issue

Reply:
Added.